namedquery/aws-athena-namedquery.json

```diff
@@ -27,6 +27,12 @@
             "minLength": 1,
             "maxLength": 262144
         },
+        "WorkGroup": {
+            "description": "The name of the workgroup that contains the named query.",
```

Comment:
we might want to be consistent with https://docs.aws.amazon.com/athena/latest/APIReference/API_CreateNamedQuery.html

Reply:
I pulled this from https://docs.aws.amazon.com/athena/latest/APIReference/API_NamedQuery.html which is the general data type doc. Since CFN is a wrapper for the data type as whole, this seemed like the best place to pull it from.

Reply:
yeah that make sense